llmfoundry/train/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you do something like this: https://github.com/mosaicml/llm-foundry/blob/main/llmfoundry/utils/__init__.py#L70 

So that we can expose `train` here?

Reply:
That way the imports can be `from llmfoundry.train import train`